### PR TITLE
workflows: Run urls-check in GitHub workflows 

### PR DESCRIPTION
**Pull request by marusak**
Also fixing bug why no po-refresh and npm-update were triggered.

Tried it on my own fork.
- Successful run of urls-check - https://github.com/marusak/cockpit/issues/33
- Successful run on npm-update - https://github.com/marusak/cockpit/pull/34

### CODE REVIEW

File: .github/workflows/npm-update.yml [hunk 1]
```diff
@@ -21,5 +21,5 @@ jobs:
 
       - name: Run npm-update bot
         run: |
-          make bots
+          tools/make-bots
```
Comment (martinpitt) [inline .github/workflows/npm-update.yml:24]: Ouch, thanks! This worked fine in starter-kit with its static Makefile, but I seem to have missed it :crying_cat_face:

File: .github/workflows/urls-check.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
+          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run urls-check action
+        run: |
+          make bots
```
Comment (martinpitt) [inline .github/workflows/urls-check.yml]: That has the same problem, though?

File: .github/workflows/urls-check.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - cron: '0 4 * * *'
+  # can be run manually on https://github.com/cockpit-project/cockpit/actions
+  workflow_dispatch:
+jobs:
+  urls-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
```
Comment (martinpitt) [inline .github/workflows/urls-check.yml:13]: This doesn't commit anything, so we don't strictly need a git config here.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! One fix necessary, then we can move this. I think we should also change urls-check to not create an issue at all if the check succeeds, that's just useless noise. It's enough if it files an issue on a broken check? (That's follow-up material, of course)

File: .github/workflows/urls-check.yml (hunk 2 above)
Comment (marusak) [reply .github/workflows/urls-check.yml]: yeah, I fix it with the other commit. Should I do this the right way in the first commit?

File: .github/workflows/urls-check.yml (hunk 3 above)
Comment (marusak) [reply .github/workflows/urls-check.yml:13]: true, I'll drop it.

**Comment by marusak** [comment]
>  I think we should also change urls-check to not create an issue at all if the check succeeds, that's just useless noise. It's enough if it files an issue on a broken check? 

`urls-check` is smart in this matter. It works like this:
- It looks for the last issue for urls-check
- If it is opened, it does nothing (waits until we fix it)
- If it is closed and the last update was more than n-days ago, it runs the check
- If the check succeeds it just comments into that closed issue that all is fine (just so we can see it ran and it bumps up the last modification [see previous point])
- If it fails, it opens a new issue (see the second point)
- (special case) If there is no opened nor closed issue and the check succeeded then create a new issue, comment all went fine and close it. This happens when we run this for the first time in a repository. And this is what you saw.

File: .github/workflows/npm-update.yml (hunk 1 above)
Comment (martinpitt) [reply .github/workflows/npm-update.yml:24]: Ah, yes please, less confusing.

File: .github/workflows/urls-check.yml (hunk 2 above)
Comment (martinpitt) [reply .github/workflows/urls-check.yml]: Ah, yes please, less confusing.

Comment (marusak) [reply .github/workflows/urls-check.yml]: done

File: .github/workflows/urls-check.yml (hunk 3 above)
Comment (marusak) [reply .github/workflows/urls-check.yml]: done

**Review by martinpitt — APPROVED** [review]
Yay, thanks!

**Pull request merged**